event_attribution_reporting_clicks.md

```diff
@@ -250,13 +251,7 @@ delete the scheduled report with the lowest priority and schedule the new report
 
 ### Multiple sources for the same trigger (Multi-touch)
 
-If multiple sources were clicked and associated with a single attribution trigger, send reports for all of them. 
-
-To provide additional utility, the browser can choose to provide additional annotations to each of these reports, attributing credits for the triggering redirect to them individually. Attribution models allow for more sophisticated, accurate measurement.
-
-The default attribution model will be last-click attribution, giving the last-clicked source for a given trigger redirect all of the credit.
```

Comment:
Let's note that by default if no priority is specified, the browser performs last-touch